include/powsybl/iidm/converter/ConversionParameters.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_CONVERSIONPARAMETERS_HPP
+#define POWSYBL_IIDM_CONVERTER_CONVERSIONPARAMETERS_HPP
+
+#include <set>
+#include <string>
+
+namespace stdcxx {
+
+class Properties;
+
+}  // namespace stdcxx
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+class Parameter;
+
+class ConversionParameters {
+public:
+    static bool readBooleanParameter(const stdcxx::Properties& parameters, const Parameter& configuredParameter);
+
+    static std::set<std::string> readStringListParameter(const stdcxx::Properties& parameters, const Parameter& configuredParameter);
```

Comment:
Sometimes the order is important: change the return type to a `std::list` or `std::vector`